pkg/module/drift/config_parser.go

```diff
@@ -175,7 +175,6 @@ func (p *Parser) interpret(cfg *BaseConfig) hcl.Diagnostics {
 			}
 		}
 	}
-	return nil
```

Comment:
Return value is always `nil` and will actually cause a panic in callers I believe

Reply:
https://go.dev/play/p/JxmTqiIB3Wj?v=goprev

Reply:
Cool, thanks for the context. Should we still remove the return value? It's not really used

Reply:
Sure.